script.js

```diff
@@ -0,0 +1,139 @@
+const toDoInput = document.querySelector('#todo_input');
+const addBtn = document.querySelector('.todo_input > p');
+const toDoList = document.querySelector('.todo_list');
+const doneList = document.querySelector('.done_list');
+const toDoLen = document.querySelector('#todo_len');
+const doneLen = document.querySelector('#done_len');
+
+const toDoSpan = toDoList.getElementsByTagName("span");
+const doneSpan = doneList.getElementsByTagName("span");
+
+const timeShow = document.querySelector('#time');
+
+function clock() {
+    let time = new Date();
+
+    let month = time.getMonth();
+    let date = time.getDate();
+    let day = time.getDay();
+    let week = ['일', '월', '화', '수', '목', '금', '토'];
+
+    let hours = time.getHours();
+    let minutes = time.getMinutes();
+    let seconds = time.getSeconds();
+
+    timeShow.innerText = 
+    `${month + 1}월 ${date}일 ${week[day]}요일 ` +
+    `${hours < 10 ? `0${hours}` : hours}:${minutes < 10 ? `0${minutes}` : minutes}:${seconds < 10 ? `0${seconds}` : seconds}`;
+        
+}
+
+// todo list 한 줄 만드는 함수
+function makeLi(value){
+    let toDoLi = document.createElement("li");
+    let toDoText = document.createElement('span');
+    let deleteBtn = document.createElement('button');
+
+    toDoText.innerText = value;
+    toDoText.addEventListener("click",moveToDo);
+
+    deleteBtn.innerText = "🗑";
+    deleteBtn.addEventListener("click",delToDo);
+
+    toDoLi.appendChild(toDoText);
+    toDoLi.appendChild(deleteBtn);
+    return toDoLi;
+}
+
+// localstorage에 저장된 데이터 불러오는 함수
+function renderLists(){
+    let toDoString = localStorage.getItem("toDoObject");
+    let toDoItems = JSON.parse(toDoString);
+    if(toDoString){
+        for(let toDoItem of toDoItems.toDo){
+            toDoList.appendChild(makeLi(toDoItem));
+        }
+        for(let doneItem of toDoItems.done){
+            doneList.appendChild(makeLi(doneItem));
+        }
+    }
+    returnLen();
+}
+
+// todo item 바뀔 때마다 localstorage에 저장하는 함수
+function storeLists(){
+    let toDoItems = [];
+    let doneItems = [];
+    for (let i = 0; i < toDoSpan.length; i++) {
+        let value = toDoSpan[i].innerText;
+        toDoItems.push(value);
+    }
```

Comment:
여기는 forEach를 사용하면 조금 더 깔끔한 코드 작성이 가능할 것 같아요
```suggestion
   toDoSpan.forEach((todoItem) => {
     toDoItems = [...toDoItems, todoItem.innerText]
     })
```

Reply:
훨씬 직관적으로 보기 좋은거같아요 forEach로 작성하는법을 익혀두겠습니다!

Reply:
이 방법이 더 직관적이네요~ 저도 하나 배워갑니다! 👍